<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinScreen</class>
 <widget class="QDialog" name="PinScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter PIN</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="text">
    <string>Please enter your PIN:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPin">
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum1">
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum2">
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum3">
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum4">
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum5">
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum6">
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum7">
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum8">
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum9">
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPinNum0">
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmPin">
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearPin">
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUndoPin">
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QLabel" name="textPinInfo">
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>